### DISCUSSION TITLE
Fehler bei Wende

### DISCUSSION THREAD

**Discussion by DarkStyler** [Courseplay/Courseplay_FS22#2364] [General]
habe ein kleines  Problem beim Wendemanöver Fahrzeug wird ständig Blockiert wenn Kurs auf Scharf eingestellt ist in Version 7.2.0.2 was kann ich dagegen tun damit das nicht mehr so weiter geht

Fehler Bilder:
![20230304182658_1](https://user-images.githubusercontent.com/95764309/222921708-42c199e8-b9b2-4db1-8741-0da303fc7716.jpg)

![20230304183332_1](https://user-images.githubusercontent.com/95764309/222921710-c47e3b5c-077b-44d2-acef-45e01b571ccf.jpg)

![20230304184421_1](https://user-images.githubusercontent.com/95764309/222921712-6b89f0e1-8549-41f7-8562-3afdec8ee1f9.jpg)

**Comment by Tensuko** [comment]
Welcher Traktor und welche Arbeitsgeräte?
Meistens passiert das, wenn die verwendeten Geräte schlecht konvertierte Mods sind, die noch die alte Kollision drin haben.

**Comment by DarkStyler** [comment]
Danke für den Hinweis lag an der der alten Kollision
